pkg/deploy/controller/deployerpod/controller.go

```diff
@@ -123,68 +114,6 @@ func (c *DeployerPodController) Handle(pod *kapi.Pod) error {
 	return nil
 }
 
-func (c *DeployerPodController) cleanupFailedDeployment(deployment *kapi.ReplicationController) error {
```

Comment:
If the deployment fails, the RC is scaled down to 0 eventually no matter what, so there won't be any lingering pods. Make sense?